tests/integration/suite/actors/metadata/shared.go

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright 2023 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implieh.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metadata
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+// In-memory state store component.
+const stateStore = `apiVersion: dapr.io/v1alpha1
```

Comment:
Consider creating a `WithActorInMemStore()` daprd process options func to prevent duplicating this here (and in other integration actor tests which all write this out).

Reply:
Done